cmd/load-smoke-data/load-smoke-data.go

```diff
@@ -38,6 +38,68 @@ func getFiles(root string) ([]string, error) {
 	return out, nil
 }
 
+func insertFileDocsToDb(ctx context.Context, fn string, catcher grip.Catcher, db *mongo.Database, logsDb *mongo.Database) {
+	var file *os.File
+	file, err := os.Open(fn)
+	if err != nil {
+		catcher.Add(errors.Wrap(err, "problem opening file"))
+		return
+	}
+	defer file.Close()
+
+	collName := strings.Split(filepath.Base(fn), ".")[0]
+	collection := db.Collection(collName)
+	// task_logg collection belongs to the logs db
+	if collName == model.TaskLogCollection {
+		collection = logsDb.Collection(collName)
+	}
+	scanner := bufio.NewScanner(file)
+	count := 0
+	for scanner.Scan() {
+		count++
+		bytes := scanner.Bytes()
+		// if the current collection is task_logg, delete from the collection the id that
+		// is about to be inserted so we can avoid dropping the collection completely.
+		if collName == model.TaskLogCollection {
+			taskLog := model.TaskLog{}
+			if err = bson.UnmarshalExtJSON(bytes, false, &taskLog); err != nil {
+				catcher.Add(errors.Wrapf(err, "problem reading document #%d from %s into TaskLog struct", count, fn))
+				continue
+			}
+			_, err := collection.DeleteOne(ctx, bson.M{"_id": taskLog.Id})
+			catcher.Add(err)
+		}
+		doc := bson.D{}
+		if err = bson.UnmarshalExtJSON(bytes, false, &doc); err != nil {
+			catcher.Add(errors.Wrapf(err, "problem reading document #%d from %s", count, fn))
+			continue
+		}
+
+		_, err := collection.InsertOne(ctx, doc)
+		catcher.Add(err)
```

Comment:
Could we expose that there is a  duplicate id error? It would make it easier to identify why an insert failed. If we accidentally try to import two documents with the same id.

Reply:
I actually see write errors written in my console when I run the script

Reply:
Oh Sweet!

Reply:
I'm assuming that if the `InsertOne` method would expose the type of error in the error it returns